mongodb/install.sls

```diff
@@ -0,0 +1,18 @@
+{% from "mongodb/map.jinja" import mongodb with context %}
+
+add_mongodb_package_repository:
+  pkgrepo.managed:
+    - name: {{ mongodb.repo }}
+    - humanname: MongoDB 2.6.5 Repository
+    - dist: {{ mongodb.repo_dist }}
+    - keyid: {{ mongodb.key }}
+    - keyserver: {{ mongodb.keyserver }}
+    - refresh_db: True
```

Comment:
I've added the repo for redhat, so I didn't end up making this optionally disabled, but I did add a condition for the dist param, which is unnecessary for redhat.